build.gradle

```diff
@@ -1,5 +1,41 @@
 import java.util.regex.Pattern
 
+plugins {
+    id 'base'
+    id 'org.jreleaser' version '1.18.0'
+}
+
+jreleaser {
```

Comment:
Maybe answered by my question below, but why is `jreleaser` needed in this top-level `build.gradle`, given it's in the conventions build file?